opus/application/static_media/js/browse.js

```diff
@@ -8,6 +8,7 @@
 /* globals MAX_SELECTIONS_ALLOWED */
 
 const infiniteScrollUpThreshold = 100;
+const galleryAndTableTooltipsDelay = 1000;
```

Comment:
curious why this delay needs to be different from the original tooltip delay

Reply:
Because that was the bug: "Also I think the tooltips on the gallery and table view show up way too fast, and often get in the way of trying to see things nearby. Can we add a little more delay?"